src/sagemaker/model.py

```diff
@@ -201,7 +204,7 @@ def _framework_env_vars(self):
             dir_name = self.uploaded_code.s3_prefix
         else:
             script_name = self.entry_point
-            dir_name = 'file://' + self.source_dir
```

Comment:
Why does this change?

Reply:
Talked offline. We need @iquintero to have a look.